yapapi/strategy/base.py

```diff
@@ -133,16 +148,25 @@ async def respond_to_provider_offer(
         if mid_agreement_payments_enabled:
             logger.debug(
                 "Enabling mid-agreement payments mechanism "
-                "as the expiration set to %ss (more than %ss).",
+                "as the expiration set to %ss (more than %ss). provider=%s",
                 expiration_secs,
                 MIN_EXPIRATION_FOR_MID_AGREEMENT_PAYMENTS,
+                provider_offer.issuer,
             )
         elif trigger_mid_agreement_payments:
             logger.debug(
                 "Expiration of %ss (more than our minimum for MAP: %ss) while negotiating "
-                "with a provider unaware of mid-agreeement-payments.",
+                "with a provider unaware of mid-agreeement-payments. provider=%s",
                 expiration_secs,
                 MIN_EXPIRATION_FOR_MID_AGREEMENT_PAYMENTS,
+                provider_offer.issuer,
+            )
+        else:
+            logger.debug(
+                "Not enabling mid-agreement payments for this activity. "
```

Comment:
but this _is_ per activity... .

Reply:
OK, but we're negotiating here and there is no "this activity" during the negotiations.
Actually, there will not be "this activity" even after the negotiations, because we're negotiating an agreement and we might have multiple activities in a single agreement (or 0 activities).

Reply:
okay, I'll update the wording

Reply:
added "planned"